sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -1,4 +1,4 @@
-﻿# Level-Zero backend specification
+﻿#Level - Zero backend specification

```

Comment:
```suggestion
﻿# Level Zero backend specification
```
The name of the specification is [Level Zero](https://spec.oneapi.io/versions/latest/elements/l0/source/index.html#l0-section). Let's change "Level - Zero" to "Level Zero".

Reply:
You need a space after the "#", otherwise this doesn't get rendered as a heading.

Reply:
reverted clang-format changes